pkg/graph/resolvers/mto_category.resolvers.go

```diff
@@ -22,8 +20,17 @@ func (r *mTOCategoryResolver) SubCategories(ctx context.Context, obj *models.MTO
 }
 
 // Milestones is the resolver for the milestones field.
-func (r *mTOSubcategoryResolver) Milestones(ctx context.Context, obj *models.MTOSubcategory) ([]*model.MTOMilestone, error) {
-	panic(fmt.Errorf("not implemented: Milestones - milestones"))
+func (r *mTOSubcategoryResolver) Milestones(ctx context.Context, obj *models.MTOSubcategory) ([]*models.MTOMilestone, error) {
+	var categoryID uuid.UUID
+	if obj.IsUncategorized() {
+		// We show everything as a subcategory, if it is uncategorized, but has a parent id, it belongs to the parent cat
```

Comment:
Took me a bit of noodling to figure out what exactly this code was doing -- took a stab at a new comment. Does this make sense?

```suggestion
	// All milestones (in the DB) have a mto_category_id, so we need to identify which to filter by in the query
	// We do this by passing the subcategory's parent ID (i.e. the Category ID) if the milestone doesn't actually have a subcategory
	// Otherwise, we can just filter by the actual subcategory ID
	var categoryID uuid.UUID
	if obj.IsUncategorized() {
```

Reply:
Thanks Clay! That sounds good. This was part of the area I was scratching my head about yesterday, I think your verbiage is clearer.